index.d.ts

```diff
@@ -179,6 +179,30 @@ export interface Options {
 	*/
 	readonly removeQueryParameters?: ReadonlyArray<RegExp | string> | boolean;
 
+	/**
+	Removes query parameters that matches specified value. May be helpful to make canonical URL.
+
+	@default undefined
+
+	@example
+	```
+	normalizeUrl('www.sindresorhus.com?foo=bar&page=1&page_size=30', {
+		removeQueryParametersByValue: [
+			{
+				key: 'page',
+				value: 1
+			},
+			{
+				key: 'page_size',
+				value: 10
```

Comment:
I realize this is to show it wouldn't apply the given URL, but removing a `page_size` of `10` makes no sense and will just confuse the user.